server/index.js

```diff
@@ -15,6 +16,13 @@ const config = require('../config/selectConfig')(argv.config);
 const initCache = require('./util/initCache');
 const logger = require('./util/errorLogger');
 
+// Initialize tracing
+if (config.server.enableDDTrace) {
+	// TODO: consider where it's useful to active plugins and do so via env configs
+	// REF: https://docs.datadoghq.com/tracing/runtime_metrics/nodejs/
+	ddTrace.init({ runtimeMetrics: true });
```

Comment:
Currently testing runtimeMetrics only in the test env. We might move the plugins object to and environment config as well...